SubmittingPatches.rst

```diff
@@ -519,3 +516,133 @@ decreasing the likelihood of your MIME-attached change being accepted.
 Exception: If your mailer is mangling patches then someone may ask
 you to re-send them using MIME.
 
+
+BACKPORTING
```

Comment:
I'd suggest to replace the all-caps headline with "Backporting", to be in line with other headlines in the documentation.

Reply:
@LenzGR All the headlines at this level are in all-caps, actually, but (a) we're talking about breaking this up into separate documents, and (b) I agree that all-caps headlines are undesirable, so I'll keep that in mind for the next round of revisions.